qcodes/instrument_drivers/Minicircuits/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .RUDAT_13G_90 import MiniCircuitsRudat13G90Usb
+from .USB_SPDT import MiniCircuitsSwitchChannelUsb, MiniCircuitsUsbSPDT
```

Comment:
Perhaps these names could be updated to make it more clear that one is a channel of the other

`MiniCircuitsSPDTUsb` and `MiniCircuitsSPDTChannelUsb` may be better names?